site/content/docs/howto/configure-concierge-supervisor-jwt.md

```diff
@@ -0,0 +1,104 @@
+
+## Prerequisites
+
+This how-to guide assumes that you have already [installed the Pinniped Supervisor]({{< ref "install-supervisor" >}}) with working ingress,
+and that you have [configured a FederationDomain to issue tokens for your downstream clusters]({{< ref "configure-supervisor" >}}).
+
+## Create a JWTAuthenticator
+
+Create a JWTAuthenticator describing how to validate tokens from your Supervisor's FederationDomain:
+
+```yaml
+apiVersion: authentication.concierge.pinniped.dev/v1alpha1
+kind: JWTAuthenticator
+metadata:
+  name: my-supervisor-authenticator
+spec:
+  # The value of the `issuer` field should exactly match the `issuer`
+  # field of your Supervisor's FederationDomain.
+  issuer: https://my-issuer.example.com/any/path
+  # You can use any `audience` identifier for your cluster, but it is
+  # important that it is unique for security reasons.
+  audience: my-unique-cluster-identifier-da79fa849
+  # If the TLS certificate of your FederationDomain is not signed by
+  # a standard CA trusted by the Concierge pods, then specify its CA
+  # as a base64-encoded PEM.
+  tls:
+    certificateAuthorityData: LS0tLS1CRUdJTiBDRVJUSUZJQ0...0tLQo=
+```
+
+If you've saved this into a file `my-supervisor-authenticator.yaml`, then install it into your cluster using:
+
+```sh
+kubectl apply -f my-supervisor-authenticator.yaml
+```
+
+Do this on each cluster in which you would like to allow users from that FederationDomain to log in.
+Don't forget to give each cluster a unique `audience` value for security reasons.
+
+## Generate a kubeconfig file
+
+Generate a kubeconfig file for one of the clusters in which you installed and configured the Concierge as described above:
+
+```sh
+pinniped get kubeconfig > my-cluster.yaml
+```
+
+This creates a kubeconfig YAML file `my-cluster.yaml`, unique to that cluster, which targets your JWTAuthenticator
+using `pinniped login oidc` as an [ExecCredential plugin](https://kubernetes.io/docs/reference/access-authn-authz/authentication/#client-go-credential-plugins).
+
+## Use the kubeconfig file
+
+Use the kubeconfig with `kubectl` to access your cluster:
+
+```sh
+kubectl --kubeconfig my-cluster.yaml get namespaces
+```
+
+You should see:
+
+- The `pinniped login oidc` command is executed automatically by `kubectl`.
+
+- Pinniped directs you to login with whatever identity provider is configured in the Supervisor, either by opening
+  your browser (for upstream OIDC Providers) or by prompting for your username and password (for upstream LDAP providers).
+
+- In your shell, you see your clusters namespaces.
+
+  If instead you get an access denied error, you may need to create a ClusterRoleBinding for username of your account
+  in the Supervisor's upstream identity provider, for example:
+
+  ```sh
+  kubectl create clusterrolebinding my-user-admin \
```

Comment:
Might be too much for a howto, but it could be cool to have an example of a group clusterrolebinding rather than an individual user.

Reply:
Thanks! Fixed in 9621ad9d2cba4cdaf7d3064a9841033696b57665.